### ISSUE TITLE
Make this repo work again

### ISSUE THREAD

**Issue by C47D** [lvgl/lv_binding_rust#184]
List of pending activities:

- [ ] Merge #165
- [ ] Merge #177 
- [ ] Merge #183 
- [ ] Discuss #176 

Sounds good @AlixANNERAUD?

**Comment by AlixANNERAUD** [comment]
Yeah that seems a good start.

**Comment by C47D** [comment]
Hi @AlixANNERAUD for the `lvgl-sys` move, should we discuss it in the other issue or here is ok?

**Comment by AlixANNERAUD** [comment]
> Hi @AlixANNERAUD for the `lvgl-sys` move, should we discuss it in the other issue or here is ok?

In the opened issue, I think. I already left comments.

**Comment by C47D** [comment]
I've added an item about cross-compilation, from the previous issues and PR's I think it stopped working somewhere. Have you tried to compile for embedded targets @AlixANNERAUD ?

**Comment by C47D** [comment]
@elfaroon I saw you are interested in LVGLv9 support

**Comment by AlixANNERAUD** [comment]
> I've added an item about cross-compilation, from the previous issues and PR's I think it stopped working somewhere. Have you tried to compile for embedded targets @AlixANNERAUD ?

Yes, I’ve also tried (and succeeded) in the past on ESP32 family. Libraries with C-to-Rust bindings are notoriously tricky to build, mainly due to linker and `libclang` path configurations in the build script when using `clang` and `bindgen`. Additionally, the build script for `lv_binding_rust`—particularly `lvgl_sys`—is quite complex, which further complicates matters.
